terracotta-kit/pom.xml

```diff
@@ -131,7 +149,7 @@ The Initial Developer of the Covered Software is
                   <addClasspath>false</addClasspath>
                 </manifest>
                 <manifestEntries>
-                  <Class-Path>ehcache/lib/ehcache-${ehcache.version}.jar ehcache/lib/slf4j-api-${slf4j.version}.jar toolkit/lib/terracotta-toolkit-runtime-${project.version}.jar</Class-Path>
+                  <Class-Path>ehcache/lib/ehcache-${ehcache.version}.jar ${slf4j.api.ehcache.classpath.path} toolkit/lib/terracotta-toolkit-runtime-${project.version}.jar</Class-Path>
```

Comment:
If we're going to go this way would it make sense to get the ehcache jar name via this mechanism too:
```xml
<executions>
  <execution>
    <id>resolve-slf4j-version</id>
    <goals>
      <goal>build-classpath</goal>
    </goals>
    <configuration>
      <includeGroupIds>net.sf.ehcache</includeGroupIds>
      <includeArtifactIds>ehcache</includeArtifactIds>
      <outputProperty>ehcache.classpath.path</outputProperty>
      <prefix>ehcache/lib</prefix>
      <pathSeparator> </pathSeparator>
    </configuration>
  </execution>
</executions>
```

Reply:
Did it